aospy/calc.py

```diff
@@ -371,24 +340,22 @@ def _get_pressure_vals(self, var, start_date, end_date, n=0):
                 **self.data_loader_attrs)
             name = self._ps_data.name
             self._ps_data = self._add_grid_attributes(
-                self._ps_data.to_dataset(name=name), 0)
+                self._ps_data.to_dataset(name=name))
             self._ps_data = self._ps_data[name]
 
             ps = self._ps_data
         if self.dtype_in_vert == 'pressure':
-            return self._get_pressure_from_p_coords(ps, name=var.name, n=n)
+            return self._get_pressure_from_p_coords(ps, name=var.name)
         if self.dtype_in_vert == internal_names.ETA_STR:
-            return self._get_pressure_from_eta_coords(ps, name=var.name, n=n)
+            return self._get_pressure_from_eta_coords(ps, name=var.name)
         raise ValueError("`dtype_in_vert` must be either 'pressure' or "
                          "'sigma' for pressure data")
 
-    def _get_input_data(self, var, start_date, end_date, n):
+    def _get_input_data(self, var, start_date, end_date):
         """Get the data for a single variable over the desired date range."""
         logging.info(self._print_verbose("Getting input data:", var))
         # If only 1 run, use it to load all data.  Otherwise assume that num
```

Comment:
Good call, will go through all the comments looking for anything else along these lines.